EDITORS.md

```diff
@@ -116,6 +136,12 @@ mason_lspconfig.setup_handlers {
 }
 ```
 
+> [!NOTE]
+> As documented under [LazyVim LSP](#lazyvim-lsp), Mason and Mason LspConfig may interfere with choosing the correct
+> version of Ruby LSP. If you're having problems with Ruby LSP starting, try adding `mason = false, cmd = {
+> vim.fn.expand("~/.rbenv/shims/ruby-lsp") }` to your `ruby_lsp.setup` table. If you're not using rbenv, this path may
+> be different.
```

Comment:
This note feels a bit short on the details and unrelated to the overall PR.

It doesn't explain why Mason may interfere with the LSP versions, why setting it to false will fix it or the relationship with version managers.

If this problem applies to NeoVim and LazyVim, then maybe we can create a separate section to discuss this and link to it. But let's do it in a separate PR.